tour-tip/test/get-tip-model.spec.js

```diff
@@ -0,0 +1,34 @@
+/*global describe,afterEach,beforeEach,it,expect*/
+
+import {default as getTipModelDefault, getById, getRandomOfSize} from '../get-tip-model';
+
+describe('tour tip get tip model', () => {
+
+	describe('getById', () => {
+
+		it('gets by ID', () => {
+			expect(getById('topics-m').id).to.equal('topics-m');
+		});
+
+		it('returns undefined if the ID does not exist', () => {
+			expect(getById('???')).to.be.undefined;
+		});
+
+		it('is the default function', () => {
+			expect(getTipModelDefault === getById).to.be.true;
+		});
+
+	});
+
+	describe('getRandomOfSize', () => {
```

Comment:
how about this approach? https://sazzer.github.io/blog/2015/08/20/Unit-Testing-ES6-Modules-Mockery/